Packs/FiltersAndTransformers/Scripts/IfElif/IfElif.py

```diff
@@ -0,0 +1,122 @@
+            | re.IGNORECASE * ('case_insensitive' in flags)
+        )
+        self.functions = {
+            'regex_match': partial(
+                re.fullmatch if 'regex_full_match' in flags else re.search,
+                flags=self.regex_flags
+            )
+        }
+        if 'case_insensitive' in flags:
+            def eq(x, y):
+                return repr(x).lower() == repr(y).lower()
+            self.operator_functions |= {
+                ast.Eq: eq,
+                ast.NotEq: lambda x, y: not eq(x, y),
+            }
+
+    def load_conditions_with_values(self, conditions, values):
+        def from_value(keys: re.Match):
+            return repr(demisto.dt(values, keys[1]))
+        conditions = re.compile('#{([\s\S]+?)}').sub(from_value, conditions)
+        self.conditions = self.evaluate(conditions)
+
+    def get_value(self, node):
+        match type(node):
+            case ast.Constant:
+                return node.value
+            case ast.List:
+                return [self.get_value(item) for item in node.elts]
+            case ast.Dict:
+                return {
+                    self.get_value(key): self.get_value(value)
+                    for key, value in zip(node.keys, node.values)
+                }
+            case ast.Name:
+                return json.loads(node.id)
+            case ast.Call:
+                return self.functions[node.func.id](*map(self.get_value, node.args))
+            case ast.Compare:
+                left = self.get_value(node.left)
+                return all(
+                    self.operator_functions[type(op)](
+                        left, left := self.get_value(right)  # noqa: F841
+                    )
+                    for op, right in zip(node.ops, node.comparators)
+                )
+            case ast.BoolOp:
+                return reduce(
+                    self.operator_functions[type(node.op)], map(self.get_value, node.values)
+                )
+            case ast.UnaryOp:
+                return self.operator_functions[type(node.op)](self.get_value(node.operand))
+            case _:
+                raise SyntaxError(
+                    f'Unsupported expression type found: {node.__class__.__name__}'
+                )
+
+    def evaluate(self, expression: str):
+        # sourcery skip: raise-from-previous-error
+        try:
+            parsed = ast.parse(expression, mode='eval')
+        except Exception:
+            raise SyntaxError(f'Cannot parse expression: {expression!r}')
+        return self.get_value(parsed.body)
+
+    def parse_conditions(self):
+        *conditions, default = self.conditions
+        result = next(
+            (
+                condition['return']
+                for condition in conditions
+                if self.evaluate(condition['condition'])
+            ),
+            default['else']
+        )
+        return result
+
+
+def main():
+    try:
+        if_elif = IfElif(**demisto.args())
```

Comment:
which argument is our conditions and which is our values? Lets unpack this list here